server/src/main/java/com/exacaster/lighter/application/sessions/SessionService.java

```diff
@@ -31,31 +34,47 @@ public class SessionService {
     private final StatementHandler statementHandler;
 
     public SessionService(ApplicationStorage applicationStorage,
-            StatementStorage statementStorage, Backend backend,
-            StatementHandler statementHandler) {
+                          StatementStorage statementStorage, Backend backend,
+                          StatementHandler statementHandler) {
         this.applicationStorage = applicationStorage;
         this.statementStorage = statementStorage;
         this.backend = backend;
         this.statementHandler = statementHandler;
     }
 
+    //TODO do we wanna return just Session or PermSession as well?
```

Comment:
we should return both, We want to show permanent sessions on the Lighter UI as we do with regular ones. This method is used by sessions API endpoint.

Reply:
done.

Reply:
Remove `//TODO` comment as well :)